submissions/tfss/description.md

```diff
@@ -0,0 +1 @@
+This is tensor flow 5 layer RNN trained neural networks using stft magnitudes and soft time masking. 
```

Comment:
Please use the template provided by us and fill out the form so that we get some basic knowledge about you and the method